app/code/Magento/Store/Console/Command/StoreListCommand.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace Magento\Store\Console\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Command\Command;
+
+/**
+ * Class StoreListCommand
+ * @package Magento\Store\Console\Command
+ */
+class StoreListCommand extends Command
```

Comment:
@antonkril Damn fine point you make there. The additions I made threw an exception when I ran `bin/magento` without any parameters at all on an uninstalled instance.

I couldn't see any precedence for conditional loading of console classes, so I rolled my own as a proof of concept. Is there a preferred way to handle this kind of thing or is my solution acceptable?

Please review, thank you.